web/apps/labelstudio/src/components/Form/Elements/ToggleRight/ToggleRight.styl

```diff
@@ -0,0 +1,56 @@
+.toggle-right
```

Comment:
@nicholasrq You mentioned SCSS is the way forward, how strongly are we taking that at this time with regards to new code? With Stylus having been long deprecated we are going to have to move all existing Stylus over to SCSS, and wanted to know if that is something we are trying to encourage in new PRs?